locales/th/php-inline.json

```diff
@@ -1,124 +1,124 @@
 {
-    "accepted": "ช่องข้อมูลนี้จะต้องรับการยอมรับ",
-    "accepted_if": "ช่องข้อมูลนี้ต้องได้รับการยอมรับเมื่อ :other มีค่าเท่ากับ :value.",
-    "active_url": "นี่ไม่ใช่ที่อยู่ URL ที่ใช้การได้",
-    "after": "นี่ต้องเป็นวันหลังจาก :date.",
-    "after_or_equal": "นี่ต้องเป็นวันหลังจากหรือเท่ากับ :date.",
-    "alpha": "ช่องข้อมูลนี้อาจจะเป็นเพียงแค่จะบรรจุจดหมาย",
-    "alpha_dash": "ช่องข้อมูลนี้อาจจะเป็นเพียงแค่จะบรรจุจดหมายของตัวเลข dashes และ underscores.",
-    "alpha_num": "ช่องข้อมูลนี้อาจจะเป็นเพียงแค่จะบรรจุจดหมายและเบอร์โทรศัพท์",
-    "array": "ช่องข้อมูลนี้จะต้องเป็นอาเรย์.",
+    "accepted": "จำเป็นต้องยอมรับฟิลด์นี้",
+    "accepted_if": "ต้องยอมรับฟิลด์นี้เมื่อ :other เท่ากับ :value",
```

Comment:
And this one.

Reply:
The sentence conveys the same meaning, but I think we can make it better by keeping the word `มีค่า` so it will be --> `"accepted_if": "ต้องยอมรับฟิลด์นี้เมื่อ :other มีค่าเท่ากับ :value`